cookbooks/ambari/recipes/ambari_ldap_sync.rb

```diff
@@ -0,0 +1,30 @@
+if node['ambari']['ambari.ldap.isConfigured'] then
+  pwd_file=node['ambari']['ldap']['password_file']
+  grp_file=node['ambari']['ldap']['groups_file']
+  cfg_dir=node['ambari']['ambari_server_conf_dir']
+  groupsFile=File.join(cfg_dir, grp_file )
+  passwordFile=File.join(cfg_dir, pwd_file)
+  ldap_pwd = node['ambari']['ldap_password'] 
```

Comment:
I believe this is a nil look-up. Am I missing the default value for `node['ambari']['ldap_password']` some where in the Ambari cookbook?